django/db/migrations/serializer.py

```diff
@@ -120,9 +120,9 @@ class EnumSerializer(BaseSerializer):
     def serialize(self):
         enum_class = self.value.__class__
         module = enum_class.__module__
-        v_string, v_imports = serializer_factory(self.value.value).serialize()
```

Comment:
We don't use `self.value.value` anymore so we don't need to serialize it, we should use name instead, i.e.
```python
v_string, v_imports = serializer_factory(self.value.name).serialize()
imports = {'import %s' % module, *v_imports}
return "%s.%s[%s]" % (module, enum_class.__name__, v_string), imports
```